test/distributed/org/apache/cassandra/distributed/test/accord/InteropTokenRangeTest.java

```diff
@@ -93,6 +93,10 @@ public void test() throws IOException
                     Assertions.assertThat(tokens(node.executeWithResult(withKeyspace("SELECT * FROM %s.tbl WHERE " + op.condition), QUORUM, pk)))
                               .describedAs("Token %d with operator %s", token, op.condition)
                               .isEqualTo(op.expected(token, tokens));
+
+                    Assertions.assertThat(tokens(node.executeWithPagingWithResult(withKeyspace("SELECT * FROM %s.tbl WHERE " + op.condition), QUORUM, 1, pk)))
```

Comment:
for this patch i won't require this, but the other patch about token ranges returning wrong data likely should... we should likely do the same thing for the `ode.executeWithResult(withKeyspace("SELECT * FROM %s.tbl WHERE " + lt.condition + " AND " + gt.condition), QUORUM, pk, pk)` logic as well